scripts/screens/ChangeGenderScreen.py

```diff
@@ -0,0 +1,631 @@
+#!/usr/bin/env python3
+# -*- coding: ascii -*-
+import os
+import pygame
+import ujson
+
+from scripts.utility import scale
+
+from .Screens import Screens
+from re import sub
+from scripts.utility import get_text_box_theme, shorten_text_to_fit
+from scripts.cat.cats import Cat
+from scripts.game_structure import image_cache
+import pygame_gui
+from pygame_gui.elements import UIWindow
+from ..game_structure.windows import PronounCreation
+from scripts.game_structure.image_button import UIImageButton, UITextBoxTweaked
+from scripts.game_structure.game_essentials import game, screen_x, screen_y, MANAGER
+from ..housekeeping.datadir import get_data_dir
+
+with open('resources/dicts/pronouns.json', 'r', encoding='utf-8') as f:
+    pronouns_dict = ujson.load(f)
+
+
+class ChangeGenderScreen(Screens):
+
+    def __init__(self, name=None):
+        super().__init__(name)
+        self.next_cat_button = None
+        self.previous_cat_button = None
+        self.back_button = None
+        self.the_cat = None
+        self.selected_cat_elements = {}
+        self.buttons = {}
+        self.next_cat = None
+        self.previous_cat = None
+        self.elements = {}
+        self.windows = None
+        self.checkboxes_text = {}
+        self.removalbuttons = {}
+        self.pronoun_template = [
+            {
+                "name": "Custom",
+                "subject": "",
+                "object": "",
+                "poss": "",
+                "inposs": "",
+                "self": "",
+                "conju": 1
+            }
+        ]
+        self.remove_button = {}
+        self.checkboxes_text = {}
```

Comment:
Since this screen no longer utilizes checkboxes, it would be good to remove all references to checkboxes.  Otherwise it becomes difficult for others to easily understand what your code is referencing.